packages/office-addin-manifest/README.md

```diff
@@ -63,3 +63,19 @@ Syntax:
 
 `manifest`: path to manifest file.
 
+Exits with a non zero error code if the manifest was invalid. Useful when running tests in your addin project.
+
+Example:
```

Comment:
I don't think the example is actually needed. There is already `npm run validate` for doing the manifest validation. I understand your point is to add manifest validation as part of the `test` script. However, because the templates already have a `validate` script, I think it is confusing to represent it this way. Since running tests and manifest validation don't need to always be done together, I think it is better to have the separate `test` and `validate` scripts where users can run them. It's probably best just to remove the example.